pom.xml

```
@@ -174,7 +172,7 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
```

Can use the httpcomponents API plugin available in Jenkins plugins and already installed by most users
```suggestion
            <groupId>org.jenkins-ci.plugins</groupId>
            <artifactId>apache-httpcomponents-client-4-api</artifactId>
```